docs/sebak_Installation.md

```diff
@@ -0,0 +1,25 @@
+## Prerequisite
```

Comment:
File name should be sebak_installation.md

Reply:
thanks :)